packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,27 @@
+# Best Practices
+The Flat select option should be used for single choice selection.
+
+The width of each of the options should be the same, that way we keep everything equal and balanced, we don't want any of the options feel more important than the other.
+
+Limit the number of options, a good rule of thumb is to have no more than five.
+
+Don't use Flat select when the effect is not immediate, consider using dropdown menu, radio buttons or checkboxes instead. Users are expect to have immediate feedback on their actions when using Flat select.
+
+If used as view switcher (tabs), make sure it makes sense hierarchically and that tabs cannot be used.
```

Comment:
I didn't knew we were doing that? I thought they were only used in form like glorified radio button :see_no_evil:

Reply:
@aiheon we do use them as glorified radio buttons... but we shouldn't!